devops/Makefile

```diff
@@ -628,4 +628,4 @@ db-backup-deploy-patroni-delete:
 # Remove redis and resources including secrets from openshift namespace.
 delete-redis:
 	@echo "+\n++ Removing redis and resources on $(NAMESPACE)\n+"
-	@oc delete -n $(NAMESPACE) all,rc,svc,dc,route,pvc,secret,configmap,sa -l app=redis
+	@oc delete -n $(NAMESPACE) all,rc,svc,dc,route,pvc,secret,configmap,sa -l app=redis
```

Comment:
Minor, no need to add this to the PR if no changes are present.

Reply:
yeah i am trying to remove it and another file devops/openshift/db-migrations-job.yml. But its keep on not saving